### PR TITLE
PvP Teams support

### PR DESCRIPTION
**Pull request by Simbiat**
PvP Teams support, fix for #89 and also slight update for Feast parser to support any Feast season, not only those registered as constants and current one.

### CODE REVIEW

File: src/Lodestone/Entities/Character/CharacterProfile.php [hunk 1]
```diff
@@ -151,6 +156,10 @@ public function getHash()
         // remove free company id, being kicked
         // should not generate a new hash
         unset($data['freecompany']);
+        
+        // remove pvp team id, being kicked
+        // should not generate a new hash
+        unset($data['pvpteam']);
```
Comment (viion) [inline src/Lodestone/Entities/Character/CharacterProfile.php]: Nice thinking!

File: src/Lodestone/Parser/Character/TraitProfile.php [hunk 2]
```diff
@@ -27,18 +27,21 @@ protected function parseProfile()
...
-                    $this->parseProfileGrandCompany($rows[3]);
+        foreach ($rows as $row) {
+            $blocktitle = $row->find('.character-block__title')->plaintext;
+            if (in_array($blocktitle, array('Race/Clan/Gender', 'Volk / Stamm / Geschlecht', 'Race / Ethnie / Sexe', '種族/部族/性別'))) {
+                $this->parseProfileRaceClanGender($row);
+            } elseif (in_array($blocktitle, array('NamedayGuardian', 'NamenstagSchutzgott', 'Date de naissanceDivinité', '誕生日守護神'))) {
+                $this->parseProfileNameDay($row);
+            } elseif (in_array($blocktitle, array('City-state', 'Stadtstaat', 'Cité de départ', '開始都市'))) {
+                $this->parseProfileCity($row);
+            } elseif (in_array($blocktitle, array('Grand Company', 'Staatliche Gesellschaft', 'Grande compagnie', '所属グランドカンパニー'))) {
```
Comment (viion) [inline src/Lodestone/Parser/Character/TraitProfile.php]: Don't suppose you could change these to php 5.4+ syntax?

```
(in_array($blocktitle, ['City-state', 'Stadtstaat', 'Cité de départ', '開始都市']))
```

**Comment by viion** [comment]
Brilliant work, got only 1 tiny nitpick :D but other than that, great!

File: src/Lodestone/Parser/Character/TraitProfile.php (hunk 2 above)
Comment (Simbiat) [reply src/Lodestone/Parser/Character/TraitProfile.php]: Did not know this syntax could be used :D Updated

**Comment by viion** [comment]
Could you throw up an issue regarding the Feast issue and a bit about it? Then we can track and look into it. For now i'll merge this as it's good stuff.

**Pull request merged**